sql/table.go

```diff
@@ -175,6 +175,76 @@ func (p *planner) getTableDesc(qname *parser.QualifiedName) (*TableDescriptor, e
 	return &desc, nil
 }
 
+// getTableLease acquires a lease for the specified table. The lease will be
+// released when the planner closes.
+func (p *planner) getTableLease(qname *parser.QualifiedName) (*TableDescriptor, error) {
+	if err := qname.NormalizeTableName(p.session.Database); err != nil {
+		return nil, err
+	}
+
+	if qname.Database() == SystemDB.Name {
```

Comment:
unless that's already somewhere, we should make sure that nobody can alter the system table.